docusaurus.config.js

```diff
@@ -92,6 +92,7 @@ const config = {
       prism: {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
+        additionalLanguages: ['java','scala'],
```

Comment:
Do we need Python here?

Reply:
no, it has inbuilt support for python, unlike scala. So, just had to add scala and java(for scala)